tests/TestExternalFile.m

```diff
@@ -157,5 +157,33 @@ function TestExternalFile_testMD5Hash(testCase)
             uuid = dj.lib.DataHash(packed_cell{1}, 'bin', 'hex', 'MD5');
             testCase.verifyEqual(uuid, '1d751e2e1e74faf84ab485fde8ef72be');
         end
+        function  TestExternalFile_test32BitRead(testCase)
```

Comment:
Missed this before, this is not really a test for an external file store. Would you move this to a new `TestBlob.m`?

Reply:
There is no `TestBlob.m` in datajoint matlab